### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by AndriiYelieva**
[DEMO](https://AndriiYelieva.github.io/react_static-list-of-posts-js/)

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.jsx [hunk 1]
```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.title}</strong>
+
+      {' by '}
+
```
Comment (IvanFesenko) [inline src/components/CommentInfo/CommentInfo.jsx:7]:
```suggestion
      <span> by </span>
```

File: src/components/PostInfo/PostInfo.jsx [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
```
Comment (IvanFesenko) [inline src/components/PostInfo/PostInfo.jsx:14]: Same here - better to wrap by tag

File: src/components/PostInfo/PostInfo.jsx [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+
+        <UserInfo user={post.user} />
```
Comment (IvanFesenko) [inline src/components/PostInfo/PostInfo.jsx:16]: User might be null so add check if user exists

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Check few comments

**Review by etojeDenys — APPROVED** [review]
awesome

**Pull request closed without merging**